src/Intent.yaml

```diff
@@ -23,6 +23,16 @@ Intent:
       description: The contexts the intent accepts. This will
         typically be a set of namespaced context types, e.g. 
         "org.symphony.contact"
+    dataReturnType: 
+      type: string
+      description: If there is a data payload associated with the intent resolution, specifies the type.  
+      This value can be either a URL to the type definition, or an alias for the type.  
+      If this value is set, the desktop agent can manage the resolution of the intent accordingly.
+    dataInputType: 
+      type: string
+      description: If there is a data payload associated with the intent, specifies the type expected.  
+      This value can be either a URL to the type definition, or an alias for the type.  
+      If this value is set, the desktop agent can manage the validation and resolution of the intent accordingly.
```

Comment:
This is a major divergence from the spec so far. We currently use the "contexts" array for the purpose of specifying the possible "input types" (plural) that an intent can accept/operate on. How do these two interact?